cmd/start.go

```diff
@@ -47,7 +52,11 @@ $ %s start 1 12      # start relaying data from specific tunnelIDs.`, appName, a
 				}
 			}
 
-			return app.Start(cmd.Context(), tunnelIDs)
+			if tunnelCreator != "" && len(tunnelIDs) != 0 {
+				return fmt.Errorf("cannot use both --tunnel-creator and --tunnel-ids flags at the same time")
```

Comment:
The --tunnel-creator and --tunnel-ids flags cannot be used together.
Please specify only one of these options at a time.